activestorage/app/controllers/active_storage/blobs/proxy_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+# Take a signed permanent reference for a blob representation and turn it into an expiring service URL for download.
+# Note: These URLs are publicly accessible. If you need to enforce access protection beyond the
+# security-through-obscurity factor of the signed blob and variation reference, you'll need to implement your own
+# authenticated redirection controller.
```

Comment:
> Take a signed permanent reference for a blob representation and turn it into an expiring service URL for download.

This doesn’t seem to match this controller’s behavior. Copy-pasted from elsewhere?